src/components/SQForm/__tests__/SQFormDropdown.test.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {screen} from '@testing-library/react';
+import SQFormDropdown from '../SQFormDropdown';
+import {renderSQForm} from '../../../utils/tests';
+import userEvent from '@testing-library/user-event';
+
+const DEFAULT_PROPS = {
+  name: 'dropdownExample',
+  label: 'Dropdown Example'
+};
+
+const DEFAULT_INTITIAL_VALUES = {
+  initialValues: {
+    dropdownExample: ''
+  }
+};
+
+const DEFAULT_OPTIONS = [
+  {label: 'Test 1', value: 1},
+  {label: 'Test 2', value: 2},
+  {label: 'Test 3', value: 3}
+];
+
+const createDropdown = (
+  options = DEFAULT_OPTIONS,
+  fieldProps = DEFAULT_PROPS
+) => {
+  return <SQFormDropdown {...fieldProps}>{options}</SQFormDropdown>;
+};
+
+it('should render list of options', () => {
+  const renderDropdown = createDropdown();
```

Comment:
Related to my above comment, the naming suggestion would also help here:

`const sqFormDropdown = renderDropdown()`